example/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '12.0'
```

Comment:
Is there a reason this requires ios 12? (it doesn't really matter as this is just the example, but i guess it should work down to ios 9?)

Reply:
https://developer.apple.com/documentation/localauthentication/logging_a_user_into_your_app_with_face_id_or_touch_id

LAContext is supported from 11.3 I think, I am trying only to support last 2 versions. But as you said this is only example app. Will try to change it to 11.3 and try it on physical device

Reply:
This was incorrect info...LAContext is supported from earlier version, however from 11.3 it allows you to specify what kind of biometry you want to use, what is available, get more error handling and messages...basically more mature version...I will try to move it to iOS10

Reply:
according to the documentation it looks like it should be supported in iOS 9: https://developer.apple.com/documentation/security/ksecuseauthenticationcontext

```
iOS 9.0+
macOS 10.11+
```